app/controllers/search_controller.rb

```diff
@@ -0,0 +1,11 @@
+class SearchController < ApplicationController
+  def index
```

Comment:
If we start adding more functionality to this action, we can refactor this to abstract away some of the functionality.